src/components/Button.tsx

```diff
@@ -219,15 +213,19 @@ const Button = (
         width: '0%',
         right: 0,
         position: 'absolute',
-        transition: 'width 500ms cubic-bezier(0.645, 0.045, 0.355, 1.000)',
+        boxShadow: 'none',
+        transition:
+          'width 500ms cubic-bezier(0.645, 0.045, 0.355, 1.000), boxShadow 500ms cubic-bezier(0.645, 0.045, 0.355, 1.000)',
```

Comment:
I think we could split this to use `transition-timing-function` and other specific properties 🥪

Also I think the `boxShadow` needs to be in css syntax `box-shadow`
```suggestion
        transition-property: 'width, box-shadow',
        transition-duration: '500ms',
        transition-timing-function: 'cubic-bezier(0.645, 0.045, 0.355, 1.000)',
```